module/Application/language/en.po

```diff
@@ -437,6 +437,10 @@ msgstr "City and Postal Code"
 msgid "Click here for approval if membership fee paid with cash."
 msgstr "Click here for approval if membership fee paid with cash."
 
+#, fuzzy
+msgid "Commisaris Verenigingsontwikkeling"
```

Comment:
```suggestion
msgid "Commissaris Verenigingsontwikkeling"
```

Reply:
Without this, you get this
![image](https://github.com/user-attachments/assets/db9a6ffc-f85e-466e-8a7c-d64fa7791de0)